Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
```

Comment:
2.x line will crash on short syntax - so it is not a matter of CS, therefore it may be worthless to run this CS fixer and waste time.
Btw, 3.x could use short array syntax and use short array syntax fixer to still keep merging simple